kubernetes/core-pod.yml

```diff
@@ -23,3 +23,10 @@ spec:
           port: 5000
         initialDelaySeconds: 10
         timeoutSeconds: 40
+      volumeMounts:
+        - name: core-volume
+          mountPath: /core/data/memory.py
```

Comment:
Please, don't mount a code, config and source data files together. It's prefer to place them into separate folders.